### ISSUE TITLE
Suggestion: Introduce antag toggle button on lobby screen

### ISSUE THREAD

**Issue by Myctai** [space-wizards/space-station-14#10828]
## Description
Introduce antag toggle button on lobby screen. For those times when you don't feel like doing the job of antagonizing people on station yet do not want to go to character preferences to toggle off antag checkboxes temporarily.

**Additional context**
Ensure state of toggle is communicated clearly, I saw some newbies get confused by how "Ready Up" works.

**Comment by lzk228** [comment]
is it really needed?

**Comment by DogZeroX** [comment]
Yes, it's a function from 13, better than ticking off everything when you don't feel like rolling antag

**Comment by lzk228** [comment]
oh, yes I remembered
firstly thought about other
